apps/ai-image-tagging/frontend/src/components/AITagView/AITagView.js

```diff
@@ -51,9 +51,9 @@ export class AITagView extends React.Component {
 
     const file = await this.props.space.getAsset(imageId);
     const locale = this.props.locale;
-    const fullURL = get(file, `fields.file.${locale}.url`);
+    const contentType = get(file, `fields.file.${locale}.contentType`);
     // test if file extension is PNG/JPEG/JPG
-    const isImageValid = new RegExp(/\.png|jpeg|jpg$/, 'i').test(fullURL);
+    const isImageValid = new RegExp(/image\/png|jpeg|jpg$/, 'i').test(contentType);
```

Comment:
why RegExp constructor instead of `/.../i`?

alternatively you'd just define valid mimetypes:

`['image/png', 'image/jpg', ...].includes(contentType.trim().toLowerCase())`

Reply:
It reads better in my opinion, i.e: testing some string against a rule. `[].includes` felt a bit awkward without writing a few more lines.